core/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -191,6 +205,14 @@ public int totalOperations() {
         return operationCounter;
     }
 
+    public long getMinSeqNo() {
```

Comment:
to be really consistent I think we need to return a tuple and read it under the lock?